src/ansi-c/cprover_builtin_headers.h

```diff
@@ -29,7 +30,8 @@ __CPROVER_bool __CPROVER_get_may(const void *, const char *);
 void __CPROVER_printf(const char *format, ...);
 void __CPROVER_input(const char *id, ...);
 void __CPROVER_output(const char *id, ...);
-void __CPROVER_cover(__CPROVER_bool condition);
```

Comment:
I think this needs to be properly deprecated if we want to remove it. There may be external users.